high-availability.helmfile.yaml

```diff
@@ -0,0 +1,94 @@
+repositories:
+  - name: dapr
+    url: https://dapr.github.io/helm-charts/
+  - name: bitnami
+    url: https://charts.bitnami.com/bitnami
+  - name: ratify
+    url: https://deislabs.github.io/ratify
+    
+releases:
```

Comment:
yeah great point! I've been looking into this and it adds a lot of resource requirements on the cluster which may be permissible for certain scenarios where HA across the board is important. Dapr installs 4 different deployments for each of it's required services. If we enable HA on that then that means we have 12 pods just for dapr. I'm looking into seeing if we can remove some the sentry and placement service of dapr since we are not using that functionality.

Reply:
spoke with @luisdlp offline about this. I think for now we can just document this and revisit it if dapr HA becomes necessary.